src/common/helpers/beautifier.ts

```diff
@@ -0,0 +1,233 @@
+    }
+    if (message.payload) {
+      message.payload = this.beautifySchema(message.payload);
+    }
+  
+    return message;
+  };
+
+  private beautifyMessages(messages: Map<string, Message>): Map<string, Message> {
+    let newMessages: Map<string, Message> = {};
+    for (const key in messages) {
+      newMessages[key] = this.beautifyMessage(messages[key]);
+    }
+    return newMessages;
+  };
+
+  private beautifyServers(servers: Server[]): Server[] {
+    return servers.map(server => {
+      server.description = this.renderMd(server.description as string);
+
+      if (server.variables) {
+        const variables = server.variables;
+        let newVariables: Map<string, ServerVariable> = variables
+
+        for (const key in variables) {
+          newVariables[key].description = this.renderMd(variables[key].description as string);
+        }
+        server.variables = newVariables; 
+      }
+
+      return server;
+    });
+  }
+
+  private beautifyTopics(topics: Map<string, Topic>): Map<string, Topic> {
+    let newTopics: Map<string, Topic> = {}
+    for (const key in topics) {
+      let topic = topics[key];
+
+      if (topic.publish) {
+        if ((topic.publish as any).oneOf) {
+          let messages: Message[] = (topic.publish as any).oneOf
+          messages = messages.map(message => this.beautifyMessage(message));
+
+          (topic.publish as any).oneOf = messages
+        } else {
+          topic.publish = this.beautifyMessage(topic.publish as Message);
+        }
+      }
+
+      if (topic.subscribe) {
+        if ((topic.subscribe as any).oneOf) {
+          let messages: Message[] = (topic.subscribe as any).oneOf
+          messages = messages.map(message => this.beautifyMessage(message));
+
+          (topic.subscribe as any).oneOf = messages
+        } else {
+          topic.subscribe = this.beautifyMessage(topic.subscribe as Message);
+        }
+      }
+
+      if (topic.parameters) {
+        topic.parameters = topic.parameters.map(param => { 
+          param.description = this.renderMd(param.description as string); 
+          return param 
+        })
+      }
+
+      newTopics[key] = topic;
+    }
+    return newTopics;
+  }
+
+  private beautifySecurity(asyncApi: AsyncApi): SecurityRequirement[] {
+    const { components, security } = asyncApi;
+    let securityRequirements: SecurityRequirement[] = [];
+
+    security!.forEach(sec => {
+      const name = Object.keys(sec)[0];
+      if (!components || !components.securitySchemes || !components.securitySchemes[name]) {
+        throw new Error(`Security definition "${name}" is not included in #/components/securitySchemes.`);
+      }
+
+      let securityComponent = components.securitySchemes[name]
+      securityComponent.description = this.renderMd(securityComponent.description as string);
+      securityRequirements.push(securityComponent);
+    })
+
+    return securityRequirements;
+  }
+
+  public beautify(asyncApi: AsyncApi): AsyncApi {
```

Comment:
It's good practice to put `public` methods on top of the class, especially if it is as big as this one